tavern_server/src/db_tables.sql

```diff
@@ -0,0 +1,676 @@
+	'Huge', 
+	'Gargantuan', 
+	'Colossal'
+);
+
+CREATE TYPE Alignment AS ENUM (
+	'Lawful Good', 
+	'Lawful Neutral', 
+	'Lawful Evil', 
+	'Neutral Good', 
+	'True Neutral', 
+	'Neutral Evil', 
+	'Chaotic Good', 
+	'Chaotic Neutral', 
+	'Chaotic Evil'
+);
+
+CREATE TYPE Attribute AS ENUM (
+	'Strength', 
+	'Dexterity', 
+	'Constitution', 
+	'Intelligence', 
+	'Wisdom', 
+	'Charisma'
+);
+
+CREATE TYPE Skill AS ENUM (
+	'Acrobatics', 
+	'Appraise', 
+	'Bluff', 
+	'Climb', 
+	'Craft', 
+	'Diplomacy', 
+	'Disable Device', 
+	'Disguise', 
+	'Escape Artist', 
+	'Fly', 
+	'Handle Animal', 
+	'Heal', 
+	'Intimidate', 
+	'Knowledge (arcana)', 
+	'Knowledge (dungeoneering)', 
+	'Knowledge (engineering)', 
+	'Knowledge (geography)', 
+	'Knowledge (history)', 
+	'Knowledge (local)', 
+	'Knowledge (nobility)', 
+	'Knowledge (planes)', 
+	'Knowledge (religion)', 
+	'Linguistics', 
+	'Perception', 
+	'Perform', 
+	'Profession', 
+	'Ride', 
+	'Sense Motive', 
+	'Sleight of Hand', 
+	'Spellcraft', 
+	'Stealth', 
+	'Survival', 
+	'Swim', 
+	'Use Magic Device'
+);
+
+CREATE TYPE SaveThrow AS ENUM (
+	'Fortitude', 
+	'Reflex',
+	'Will'
+);
+
+CREATE TYPE CharacterStat AS ENUM (
+	'Name',
+	'Race',
+	'Size',
+	'Height',
+	'Weight',
+	'Age',
+	'Gender',
+	'Alignment',
+	'Deity',
+	'Languages',
+	'Appearance'
+);
+
+CREATE TYPE CombatStat AS ENUM (
+	'Melee Attack Bonus',
+	'Ranged Attack Bonus',
+	'CMB',
+	'CMD',
+	'Armor Class',
+	'Touch AC',
+	'Flat-Footed AC',
+	'Initiative Bonus',
+	'Damage Reduction',
+	'Spell Resistance',
+	'Speed',
+	'Fortititude Save',
+	'Reflex Save',
+	'Will Save'
+);
+
+CREATE TYPE RaceType AS ENUM (
+	'Abberation', 
+	'Animal', 
+	'Construct', 
+	'Dragon', 
+	'Fey', 
+	'Humanoid', 
+	'Magical Beast', 
+	'Monstrous Humanoid', 
+	'Ooze', 
+	'Outsider', 
+	'Plant', 
+	'Undead', 
+	'Vermin'
+);
+
+CREATE TYPE CasterType AS ENUM (
+	'Spontaneous', 
+	'Prepared'
+);
+
+CREATE TYPE MagicSchool AS ENUM (
+	'Abjuration', 
+	'Conjuration', 
+	'Divination', 
+	'Enchantment', 
+	'Evocation', 
+	'Illusion', 
+	'Necromancy', 
+	'Transmutation'
+);
+
+CREATE TYPE SpellRange AS ENUM (
+	'Personal',
+	'Touch',
+	'Close',
+	'Medium',
+	'Long',
+	'Unlimited'
+);
+
+CREATE TYPE WeaponClass AS ENUM (
+	'Axes', 
+	'Heavy Blades', 
+	'Light Blades', 
+	'Bows', 
+	'Close', 
+	'Crossbows', 
+	'Double', 
+	'Firearms', 
+	'Flails', 
+	'Hammers', 
+	'Monk', 
+	'Natural', 
+	'Polearms', 
+	'Siege Engines', 
+	'Spears', 
+	'Thrown', 
+	'Tribal'
+);
+
+CREATE TYPE ArmorClass AS ENUM (
+	'Light', 
+	'Medium', 
+	'Heavy'
+);
+
+CREATE TYPE EquipmentSlot AS ENUM (
+	'None',
+	'Armor', 
+	'Belts', 
+	'Body', 
+	'Chest', 
+	'Eyes', 
+	'Feet', 
+	'Hands', 
+	'Head', 
+	'Headband', 
+	'Neck', 
+	'Ring (left)', 
+	'Ring (right)', 
+	'Shield', 
+	'Shoulders', 
+	'Wrist'
+);
+
+CREATE TYPE DamageType AS ENUM (
+	'Bludgeoning', 
+	'Slashing', 
+	'Piercing', 
+	'Energy', 
+	'Acid', 
+	'Fire', 
+	'Electricity', 
+	'Cold', 
+	'Sonic', 
+	'Positive', 
+	'Negative', 
+	'Nonlethal'
+);
```

Comment:
Maybe these should be separated into multiple categories? A sword enchanted with fire could do slashing and fire damage, right?

Reply:
That could work. I've changed the damage type column on a weapon into an array, so a sword can have multiple types of damage. Also worth noting, in the table Weapons, damage has been made an array of TEXT items (TEXTis a placeholder for dice shorthand, IE 2d6)